samples/core/QueryFilters/QueryFilters.csproj

```
@@ -5,7 +5,7 @@
     <RootNamespace>Samples</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.0.0-rc1.19456.14" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.0.0-rc1.19456.10" />
```

should you update packages like these as well?

```suggestion
    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.0.0" />
```

---

Absolutely, thanks! Corrected these (and some others) and rebased.